docs/docs/product-documentation.md

```diff
@@ -150,15 +153,54 @@ Figure 2: Product feature diagram (Swapchain, 2020c) [5]
 
 ### 3.2. Tutorials
 
-***
+//sollen wir hier am Ende einen Screencast einfügen, in dem man sieht, wie man die App bedienen muss? 
 
 ### 3.3. User guides
 
-***
+#### 3.3.1 How do I get started? 
+
+To trade cryptocurrency with Swapchain no user account is needed. You just enter the app and provide the needed information. 
+
+#### 3.3.2 What information do I need to trade? 
+
+- Your crpytocurrency address: Alphanumeric characters representing the destination for a payment.
+- Your public key: Part of the keys used for encryption and decryption of information. The public key enables message encryption only when the respective private key is used. Is of no value without the respective private key.
+- Your private key: Secret key that should not be revealed to anybody else then the key owner.
+
+#### 3.3.3 What parameters can I set? 
+
+- Bid Amount: The amount of currency you want to give away. 
+- Ask Amount: the amount of currency you want to receive. 
+- Bid Type: The type of cryptocurrency you want to give away (BTC/BTS). 
+- Ask Type: The type of cryptocurrency you want to receive (BTC/BTS). 
+- Transaction fee priority: Transaction fees you want to pay for your order to be processed. The lower the transaction fee the longer the transaction will take (Low/Medium/High priority and transaction fees respectively).
+- Timelock: The duration of the timelock being active on the HTLC. As soon as the timelock expires the currency from the pending transaction is refunded to the owner (Low/Medium/High duration of timelock). 
+
+#### 3.3.4 How do I initiate the swap? 
+
+1. Fill out the form with all the needed information 
```

Comment:
Enter the swapchain app and choose the Proposer option in order to initiate a swap.